modules/message-lane/README.md

```diff
@@ -0,0 +1,132 @@
+# Message Lane Module
```

Comment:
I wanna start off by saying that I really appreciate the effort you put into this document. I think it'll be super useful to anybody that comes along and wants to understand the complexities of the `message-lane` benchmarking.

However, I do think there is a lot of room for improvement in terms of the way it is written. 

1. Keeping line lengths within our specified `.editorconfig` guidelines

I'm pretty sensitive to this since I read basically everything I can inside of my terminal instead of hopping on GitHub or whatever - so I might be a little bias, but hear me out. Even using my full monitor's width there are lines I can't read without scrolling to the right. I would expect that most people that want to read this document will not be doing so from GitHub, but will instead be looking at it from their code editors (probably alongside the actual benchmarking code). Having it be legible in this way would (in my opinion) be a nice quality of life improvement.

2. Moving longer descriptions and formulas out of tables

As a way to partially mitigate (1) I would recommend that we move some content out of tables and into standalone paragraphs or sections. This would make the content more legible in general. As a concrete example, consider the description for `ExpectedProofSize`. It's a good description, but when rendered it is squished within the tiny table box making it hard to read.

3. A section on how to run, interpret, and update weights (could be done in a different PR)

Part of the key of benchmarks is that they're able to adapt given a changing environment. In order to achieved that we need a detailed, reproducible, and somewhat dummy-proof set of instructions that others can use. Even if we link to the Substrate benchmarking doc and add a few sentences I think that would be better than having nothing.

I do have to admit I haven't dove into the _content_ of the doc yet, but I will read it carefully in the coming days :).

Reply:
@svyatonik Why'd you remove the README file and put it in the PR description? I think the code is the right place to have the explanations of how the weights are being computed. It just needed some "beautifying"

Reply:
Sorry - I was in a hurry yesterday, so haven't left an explanation. In brief: I don't think we have a time for "beautifying" readmes right now && I do not think we have a time for arguing about whether readme is "beautiful" enough or not. So let it lay in PR description for now, until we'll have some spare time for working on beautiful readmes.

Reply:
Personally, I think this is one of those cases where the upfront work will pay dividends in the future. I'd rather have this done right now, even if time is a bit tight, than in 6 months when none of us will remember what magic went into the benchmarking.

At the very least I think this should be part of the codebase where it's easily accessible to those working on benchmarking, even if it's not quite at the cleaned up state where I think it should be (i.e the improvements from my first comment).